Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Suspicious_Powershell_Command_Execution.yml

```diff
@@ -0,0 +1,1479 @@
+                  simple: alert.targetprocesscmd
+                iscontext: true
+              raw: {}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 507,
+          "y": -304
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: affd416a-2c19-4c4d-8fa6-acffd083b3a2
+    type: regular
+    task:
+      id: affd416a-2c19-4c4d-8fa6-acffd083b3a2
+      version: -1
+      name: Get Indicators Reputation
+      description: commands.local.cmd.enrich.indicators
+      script: Builtin|||enrichIndicators
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      indicatorsValues:
+        complex:
+          root: CommandLineAnalysis.analysis.original
+          accessor: indicators
+          transformers:
+          - operator: AppendIfNotEmpty
+            args:
+              item:
+                value:
+                  simple: CommandLineAnalysis.analysis.decoded.indicators
+                iscontext: true
+              raw: {}
+          - operator: uniq
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 507,
+          "y": 210
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 12eb0eb8-fb5d-4c90-82ac-5d65610dc9ba
+    type: condition
+    task:
+      id: 12eb0eb8-fb5d-4c90-82ac-5d65610dc9ba
+      version: -1
+      name: Check Commandline Risk Score
+      description: Evaluate the risk score from CommandLineAnalysis to determine if further investigation or remediation is needed.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "2"
+      Malicious:
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: Malicious
+      condition:
+      - - operator: greaterThanOrEqual
+          left:
+            value:
+              simple: CommandLineAnalysis.score
+            iscontext: true
+          right:
+            value:
+              simple: "40"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 507,
+          "y": -110
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: b2420736-f04f-4594-8f41-e809e5899d57
+    type: title
+    task:
+      id: b2420736-f04f-4594-8f41-e809e5899d57
+      version: -1
+      name: Remediation
```

Comment:
Let's find a way to remove one of the 'Remediation' section headers. Having them two might seem like a bug/mistake.